docs/tutorials/custom_images.md

```diff
@@ -8,47 +8,24 @@ By default, Fae includes a thumbnail size called `thumb` that is triggered like
 = image_tag(item.image.asset.thumb.url)
 ```
 
-You may copy the contents of the `uploaders/fae/image_upload.rb` file and add your own options, such as mobile and tablet and call them the same way.
+If you need to add your own image versions to Carrierwave you can copy the contents of https://github.com/wearefine/fae/blob/master/app/uploaders/fae/image_uploader.rb to `app/uploaders/fae/image_uploader.rb` of the parent application. From there you can add any custom image proccessing you like.
+
+The caveat being any bugfixes or features added to this file in Fae will now be ignored in favor of your local file. If you are updating Fae, you may want to verify this base file hasn't changed.
+
+In this example we will be adding some extra versions to support a responsive design:
 
 ```ruby
-# encoding: utf-8
-module Fae
-  class ImageUploader < CarrierWave::Uploader::Base
-    include CarrierWave::RMagick
```

Comment:
This line ended up being my issue. RMagick needs to be MiniMagick to match up to what Fae is currently configured to use.